AnkiDroid/src/test/java/com/ichi2/utils/AssetHelperTest.kt

```diff
@@ -13,20 +13,18 @@
  You should have received a copy of the GNU General Public License along with
  this program.  If not, see <http://www.gnu.org/licenses/>.
  */
+package com.ichi2.utils
 
-package com.ichi2.utils;
-
-import org.junit.Test;
-
-import static org.hamcrest.CoreMatchers.is;
-import static org.hamcrest.MatcherAssert.assertThat;
-
-public class AssetHelperTest {
+import com.ichi2.utils.AssetHelper.guessMimeType
+import org.hamcrest.CoreMatchers
+import org.hamcrest.MatcherAssert
+import org.junit.Test
 
+class AssetHelperTest {
     @Test
-    public void guessMimeTypeTest() {
-        assertThat(AssetHelper.guessMimeType("test.txt"), is("text/plain"));
-        assertThat(AssetHelper.guessMimeType("test.png"), is("image/png"));
-        assertThat(AssetHelper.guessMimeType("test.zip"), is("application/zip"));
+    fun guessMimeTypeTest() {
```

Comment:
Can you add this and re-push the second commit?

```suggestion
    @KotlinCleanup("is --> equalTo")
    fun guessMimeTypeTest() {
```

Reply:
Okay, but what is it for ?